BitwardenShared/UI/Auth/Landing/LandingView.swift

```diff
@@ -67,68 +68,81 @@ struct LandingView: View {
     var scrollingContent: some View {
-        ScrollView {
-            VStack(alignment: .leading, spacing: 16) {
-                Image(decorative: Asset.Images.logo)
-                    .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
-                    .frame(maxWidth: .infinity)
-                    .padding(.vertical, 45)
-
-                Text(Localizations.loginOrCreateNewAccount)
-                    .styleGuide(.title2)
-                    .multilineTextAlignment(.center)
-                    .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
-                    .frame(maxWidth: .infinity)
-
-                BitwardenTextField(
-                    title: Localizations.emailAddress,
-                    text: store.binding(
-                        get: \.email,
-                        send: LandingAction.emailChanged
-                    ),
-                    accessibilityIdentifier: "EmailAddressEntry"
-                )
-                .textFieldConfiguration(.email)
-                .onSubmit {
-                    guard store.state.isContinueButtonEnabled else { return }
-                    Task { await store.perform(.continuePressed) }
-                }
-
-                RegionSelector(
-                    selectorLabel: Localizations.loggingInOn,
-                    regionName: store.state.region.baseUrlDescription
-                ) {
-                    await store.perform(.regionPressed)
-                }
-
-                Toggle(Localizations.rememberMe, isOn: store.binding(
-                    get: { $0.isRememberMeOn },
-                    send: { .rememberMeChanged($0) }
-                ))
-                .accessibilityIdentifier("RememberMeSwitch")
-                .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
-                .toggleStyle(.bitwarden)
-
-                AsyncButton(Localizations.continue) {
-                    await store.perform(.continuePressed)
-                }
-                .accessibilityIdentifier("ContinueButton")
-                .disabled(!store.state.isContinueButtonEnabled)
-                .buttonStyle(.primary())
-
-                HStack(spacing: 4) {
-                    Text(Localizations.newAroundHere)
-                        .foregroundColor(Asset.Colors.textSecondary.swiftUIColor)
-                    Button(Localizations.createAccount) {
-                        store.send(.createAccountPressed)
+        GeometryReader { geometry in
+            ScrollView {
+                VStack(spacing: 0) {
+                    Spacer()
+
+                    Image(decorative: Asset.Images.logo)
+                        .resizable()
+                        .scaledToFit()
+                        .foregroundColor(Asset.Colors.iconSecondary.swiftUIColor)
+                        .frame(maxWidth: .infinity, maxHeight: 34)
+                        .padding(.horizontal, 12)
+
+                    Spacer()
```

Comment:
🎨 I wonder if we need to add some minimum height to these Spacers() (or padding on the image) as the content below grows? Could you check with design?

<img width="568" alt="Screenshot 2024-11-01 at 9 30 33 AM" src="https://github.com/user-attachments/assets/c598bdb2-47bc-464f-bb3b-be87482e90e5">

Reply:
It looks like the designs have 24 points of padding above and below that top image on other screens which don't have extra space at the top.

Reply:
updated!